espnet2/lm/espnet_model.py

```diff
@@ -1,4 +1,5 @@
 from typing import Dict
```

Comment:
ditto
Please increase the test coverage.

Reply:
fixed. Now it's coveraged by ci/test_integration.sh.